.github/workflows/main.yml

```diff
@@ -0,0 +1,45 @@
+name: main
```

Comment:
Could we rename this workflow to `pr` and `pr.yml`? I think that's a more clear name since this doesn't really related to `main` in any way.

Reply:
Yeah renamed file to `pr.yml` and name: `Pull Request` prefer having a full clarifying name at the top if you don't mind